Tools/release-clipper.js

```diff
@@ -76,6 +77,12 @@ async function main() {
 		console.info(await execCommand('mv ' + dist.name + '.zip ..'));
 	}
 
+	console.info('Creating source tarball for code validation...');
+	process.chdir(clipperDir + '/../');
```

Comment:
The current dir is changed here, but not changed back to what it was afterwards. Doesn't it mean that the commands that follow will be run from the wrong dir?

Reply:
No other commands are run, except the `git` commands, which can be run from any directory within the repo.

Reply:
Let me re-phrase: The commands I added require the working dir to be in that directory. At least the last 2 commands (1. creating the zip, 2. moving it, and deleting the temp dir).

The term "no other commands" refers to commands that were already in the script (the git commands).

Reply:
You're right, it won't affect the git commands afterwards so this is fine.